src/hacks/ESP.cpp

```diff
@@ -1506,7 +1598,7 @@ void _FASTCALL ProcessEntity(CachedEntity *ent)
                                         AddEntityString(ent, charge + "% Uber", colors::Health(floor(CE_FLOAT(weapon, netvar.m_flChargeLevel) * 100), 100));
                                     }
                                     else
-                                        AddEntityString(ent, charge + "% Uber | Charges: " + std::to_string(floor(CE_FLOAT(weapon, netvar.m_flChargeLevel) / 0.25f)), colors::Health((CE_FLOAT(weapon, netvar.m_flChargeLevel) * 100), 100));
+                                        AddEntityString(ent, charge + "% Uber | Charges: " + std::to_string((int) floor((CE_FLOAT(weapon, netvar.m_flChargeLevel) / 0.25f) * 100) / 100), colors::Health((CE_FLOAT(weapon, netvar.m_flChargeLevel) * 100), 100));
```

Comment:
```suggestion
                                        AddEntityString(ent, charge + "% Uber | Charges: " + std::to_string((int) floor(CE_FLOAT(weapon, netvar.m_flChargeLevel) / 0.25f), colors::Health((CE_FLOAT(weapon, netvar.m_flChargeLevel) * 100), 100));
```

This makes more sense IMO